vault/core.go

```diff
@@ -120,7 +120,8 @@ var (
 	LastRemoteWAL                = lastRemoteWALImpl
 	LastRemoteUpstreamWAL        = lastRemoteUpstreamWALImpl
 	WaitUntilWALShipped          = waitUntilWALShippedImpl
-	storedLicenseCheck           = storedLicenseCheckImpl
+	storedLicenseCheck           = func(c *Core, conf *CoreConfig) error { return nil }
+	LicenseAutoloaded            = func(*Core) bool { return false }
```

Comment:
Why make this a standalone function that accepts Core instead of a function on Core like in the ENT PR? Isn't that going to conflict when we merge OSS -> ENT?

Reply:
So we can do our assign-func-var technique to call a stub in OSS.